Add attributes for publicatios extend metadata

cuent opened the pull request:
I notice when publications are published in a journal, there are other attributes that are not in the docs (_BV, BT, PB_).  I don't know exactly what means each attribute, but I added some description;  they are mostly related with journals meta data.

Here is an example of the extended meta data for this [publication](https://academic.microsoft.com/#/detail/992652367). This is the query executed. 

`
https://westus.api.cognitive.microsoft.com/academic/v1.0/evaluate?expr=Id=992652367&attributes=Id,Ti,L,Y,D,CC,ECC,AA.AuN,AA.AuId,AA.AfN,AA.AfId,AA.S,F.FN,F.FId,J.JN.J.JId,C.CN,C.CId,RId,W,E&
`

{"BV":"The Journal of Academic Librarianship","BT":"a","PB":"JAI"}

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

cuent committed:
Add attributes for publicatios extend metadata

I notice when publications are published in a journal, there are other attributes that are not in the docs (_BV, BT, PB_).  I don't know exactly what means each attribute, but I added some description;  they are mostly related with journals meta data.

Here is an example of the extended meta data for the publication https://academic.microsoft.com/#/detail/992652367.

{"BV":"The Journal of Academic Librarianship","BT":"a","PB":"JAI"}
